javascript-conditionals.tar/javascript-conditionals/main.js

```diff
@@ -0,0 +1,109 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return 'true';
+  } else {
+    return 'false';
+  }
+}
+console.log('isUnderFive:', isUnderFive(3));
+
+function isEvenNumber(number) {
+  if (number % 2 === 0) {
+    return 'true';
+  } else {
+    return 'false';
+  }
+}
+console.log('is Even Number:', isEvenNumber(2));
+
+function startsWithJ(string) {
+  if (string === 'J'[0]) {
+    return 'true';
+  } else {
+    return 'false';
+  }
+}
+console.log('startsWithJ:', startsWithJ('j'));
+
+function isOldEnoughToDrink(person) {
+  if (person >= 21) {
+    return 'true';
+  } else {
+    return 'false';
+  }
+}
+console.log('isOldEnoughToDrink:', isOldEnoughToDrink(21));
+
+function isOldEnoughToDrive(person) {
+  if (person >= 16) {
+    return 'true';
+  } else {
+    return 'false';
+  }
+}
+console.log('isOldEnoughToDrive:', isOldEnoughToDrive(15));
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person >= 0) {
```

Comment:
Interesting way to write it ;P
Though since we all know the `else` statement should never get run - it'd be better not to include it (What if an input was said where `person.age = -1` as a bug? It would give you an incorrect answer of `true`!)